packages/api/src/votingStrategies/linearQuadraticFunding.ts

```diff
@@ -32,35 +31,66 @@ export const fetchVotesHandler = async (
           from
           to
         }
+        round {
+          roundStartTime
+          roundEndTime
+          token
+        }
       }
     }
   `;
 
   // fetch from graphql
   const response = await fetchFromGraphQL(chainId, query, variables);
 
-  const votes = response.data?.votingStrategies[0]?.votes;
+  const votes = response.data?.votingStrategies[0]?.votes ?? [];
+  const round = response.data?.votingStrategies[0]?.round;
 
   let contributions: QFContribution[] = [];
 
-  votes.map((vote: any) => {
-    const contribution = {
-      projectId: vote.to, // TODO: we will have to update this to project id eventually
-      contributor: vote.from,
-      amount: Number(vote.amount),
-      token: vote.token,
-    } as QFContribution;
+  const { error } = getChainName(chainId);
 
-    contributions.push(contribution);
-  });
+  if (error) {
+    throw new Error(`ChainId ${chainId} is not supported by CoinGecko's API.`);
+  }
+
+  contributions = await Promise.all(
+    votes.map(async (vote: any) => {
+      // denominate the contribution in the round token
+      // TODO: look into this logic. When should conversion happen?
+      // TODO: Determine how we want to pass any errors. If the conversion is unavailable, then the converted amount will
+      //       be the same as the original amount. Is this an error and should it be reported to the user?
+      //       The DenominationResponse type is defined in types.ts and includes a message/success field for future use.
+
```

Comment:
^ could we create follw up story on this as well so that we can get them added into our backlog